src/client/sandbox/node/document/index.js

```diff
@@ -77,6 +81,14 @@ export default class DocumentSandbox extends SandboxBase {
 
             const result = nativeMethods.documentOpen.apply(document, args);
 
+            // NOTE: Chrome does not remove the "%hammerhead%" property from window
```

Comment:
// NOTE: Chrome does not remove the "%hammerhead%" property from window
// after document.write call